itests/src/test/resources/testconfiguration.properties

```diff
@@ -1247,3 +1247,8 @@ erasurecoding.only.query.files=\
   erasure_commands.q,\
   erasure_explain.q,\
   erasure_simple.q
+
+# tests that requires external database connection
+externalDB.llap.query.files=\
+  dataconnector.q,\
```

Comment:
Where is the dataconnector.q test?

Reply:
dataconnector.q test is a newly created qtest in this PR[HIVE-24396](https://github.com/apache/hive/pull/2037/). It is used to test dataconnector feature, which may require external database launch in the near future